<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RepoManConfigPage</class>
 <widget class="QWidget" name="RepoManConfigPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="grpEnableAutoRefresh">
     <property name="title">
      <string>Automatic Refresh enabled</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkGitRefresh">
        <property name="text">
         <string>Git object refresh interval</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spnGitRefresh">
        <property name="specialValueText">
         <string>Never</string>
        </property>
        <property name="suffix">
         <string> seconds</string>
        </property>
        <property name="minimum">
         <number>29</number>
        </property>
        <property name="maximum">
         <number>120000</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkWorktreeRefresh">
        <property name="text">
         <string>Working tree refresh interval</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spnWorktreeRefresh">
        <property name="specialValueText">
         <string>Never</string>
        </property>
        <property name="suffix">
         <string> seconds</string>
        </property>
        <property name="minimum">
         <number>29</number>
        </property>
        <property name="maximum">
         <number>120000</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkIndexRefresh">
        <property name="text">
         <string>Index refresh interval</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spnIndexRefresh">
        <property name="specialValueText">
         <string>Never</string>
        </property>
        <property name="suffix">
         <string> seconds</string>
        </property>
        <property name="minimum">
         <number>29</number>
        </property>
        <property name="maximum">
         <number>120000</number>
        </property>
        <property name="value">
         <number>300</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>154</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
